ehrql/tables/tpp.py

```diff
@@ -552,6 +553,157 @@ class clinical_events_ranges(EventFrame):
     )
 
 
+@table
+class covid_therapeutics(EventFrame):
+    """
+    The COVID Therapeutics dataset contains information on COVID treatments used in inpatient
+    and outpatient settings.
+
+    **Metadata**
+
+    * **Data provider** NHS England
+    * **Participation / Coverage** Inpatients and outpatients treated with antivirals/nMABs for COVID-19 in England
+    * **Provenance** Data sourced largely from BlueTeq system (forms completed by clinicians)
+    * **Update frequency in OpenSAFELY** Approximately weekly
+    * **Delay between event occurring and event appearing in OpenSAFELY** Approximately 2-9 days
+    * **Collected information** Treatment start date; therapeutic intervention; COVID indication, current status, risk group, region
+
+
+    **Overview**
+
+    Antivirals and neutralising monoclonal antibodies (nMABs) for COVID-19 can be
+    administered in inpatient setting or, for outpatients, in COVID Medicine Delivery
+    Units (CMDUs) specifically set up for this purpose. For patients considered for
+    these treatments, clinicians submit completed forms to NHS England. Each row
+    represents one completed form for one course of treatment. Data received by
+    OpenSAFELY currently covers patients who were approved for treatment. The patient
+    may or may not have actually received the treatment or completed the course (but we
+    assume that they usually do). They may have another form completed for another
+    treatment, either because it was decided to give them a different treatment, or for
+    some other reason. They may in theory also have another form completed some months
+    later for another instance of infection.
+
+    Treatment dates may be in the past or future at the point when the form is
+    submitted.
+
+    Note that this dataset contains some **duplicate** rows – some full duplicates and
+    some partial duplicates.
```

Comment:
It shouldn't contain any fully duplicate rows; it will contain more than 1 row per patient, but mostly those should be valid - different visits/treatments. It probably does contain some partial duplicates.

Reply:
sorry, yes, updated to this, wdyt?
```
    Note that this dataset may contain **duplicate** rows – full duplicates are removed
    but there may remain some partial duplicates.
```